docs/_templates/2019/base.html

```diff
@@ -112,7 +112,7 @@ <h3>Join us now!</h3>
 
```

Comment:
in [Line 73](https://github.com/writethedocs/www/blob/ee7987e99153c135916c6be937e981deff503d2c/docs/_templates/2019/base.html#L73), is there a way to revise this conditional to also exclude the hike page?

When I tried the hike page for this PR, I looked at the links and thought "Oh, so that's how I sign up for the hike".

Reply:
This will be removed because the tickets will be removed from sale.